mitodl/odl-video-service
# Paginate videos on collection detail page.

#### What are the relevant tickets?
Closes #503.

#### What's this PR do?
1. Adds ersatz pagination to videos on collection detail pages. See [my comment here](https://github.com/mitodl/odl-video-service/blob/dorska/ersatz_video_pagination/static/js/components/PaginatedVideoList.js#L3) for why I say 'ersatz'.

1. Adjusts styling class names for paginate video lists.

1. Fixes `Paginator` styling

1. Consolidates pagination type definitions for both video pagination and collections pagination.

#### How should this be manually tested?
1. Make a collection with a bunch of videos. For example, with some shell commands like this:
  ```
import ui.factories
collection = ui.factories.CollectionFactory()
collection.save()
for i in range(100): ui.factories.VideoFactory(collection=collection)
  ```
2. Go to the detail page for the collection. You should see a paginated list that looks reasonable.
3. Create a collection with no videos and view its detail page. It should look reasonable.
